packages/node_modules/@webex/common/src/patterns.js

```diff
@@ -15,7 +15,7 @@ export default {
    * @todo add a better email address matcher
    * @type {RegExp}
    */
-  email: /^[^\s]+?@[^\s]+?$/,
+  email: /[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*/,
```

Comment:
I might need to brush up on my RegEx but, are `!#$%&'` allowed on an email address?
Also, I'd like to see character classes when possible to simplify the expression

Reply:
These were pulled from a w3 reference. I've updated the regex to use what Taymoor provided and included details in the JSDoc in regards to the reference.